tool/ci.dart

```diff
@@ -98,12 +99,12 @@ void main(List<String> arguments) async {
         'pkgs/hooks/example/build/native_add_app/',
       ),
       'dart',
-      ['--enable-experiment=native-assets', 'build', 'bin/native_add_app.dart'],
+      ['--enable-experiment=native-assets', 'build', 'cli', 'bin/native_add_app.dart'],
     );
     _runProcess(
       repositoryRoot
           .resolve(
-            'pkgs/hooks/example/build/native_add_app/bin/native_add_app/native_add_app.exe',
+            'pkgs/hooks/example/build/native_add_app/build/cli/${Abi.current().toString().replaceAll('_', '-')}/bundle/bin/native_add_app',
```

Comment:
We need a `.exe` for Windows!

https://github.com/dart-lang/sdk/issues/60946#issuecomment-2981020715

Reply:
Interesting that the binary starts executing correctly without the `.exe` and only fails to load the lib...

Thanks for looking into this!

Reply:
Apparently this is default Windows behavior based on `PATHEXT`.

Yeah, I'm debugging why it's failing on the Windows machine.